lib/bundles/inspec-init/profile.rb

```diff
@@ -0,0 +1,69 @@
+module Init
+  class Profile
```

Comment:
I'd suggest the name `ProfileGenerator`, as it is more in line with the meaning.  Or `ProfileTemplater`.